alf/examples/muzero_repr_conf.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from alf.optimizers.adam_tf import AdamTF
```

Comment:
Should use the `alf.optimizers.AdamTF` instead. `adam_tf.AdamTF` is modified in alf.optimizers.optimizers.py

Reply:
Ah results from blindly trust the auto-importer ... Thanks! Fixed both occurences.